dbusmock/templates/bluez5.py

```diff
@@ -92,18 +106,22 @@ def AddAdapter(self, device_name, system_name):
         'PairableTimeout': dbus.UInt32(180, variant_level=1),
     }
 
+    if device_name == 'hci1':
+        adapter_properties['Address'] = 'AA:01:02:03:04:05'
+
     self.AddObject(path,
                    ADAPTER_IFACE,
                    # Properties
                    adapter_properties,
                    # Methods
                    [
                        ('RemoveDevice', 'o', '', ''),
-                       ('StartDiscovery', '', '', ''),
+                       ('StartDiscovery', '', '', DeviceDiscovery),
```

Comment:
As mentioned, you probably want to disable `Discovering` by default, because this is what this changes.